src/main/java/com/eerussianguy/blazemap/gui/MinimapConfigurator.java

```diff
@@ -0,0 +1,185 @@
+package com.eerussianguy.blazemap.gui;
```

Comment:
wrong package, this is not a generic ui component, shouldn't be on the gui package.
needs to be on feature/maps

Reply:
- [ ] Put `MinimapConfigurator` in more relevant package